### ISSUE TITLE
Adjust Learning rate 

### ISSUE THREAD

**Issue by Samjith888** [facebookresearch/detectron2#938]
In detectron2, the learning rate can be reduced by GAMMA through the iterations 
`cfg.SOLVER.STEPS`

How to increase the learning rate at specific iteration by using detectron2 ?

**Comment by ppwwyyxx** [comment]
See pytorch scheduler https://pytorch.org/docs/stable/optim.html#how-to-adjust-learning-rate
